public/pages/workflows/components/workflow_list.tsx

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React, { useState } from 'react';
+import { EuiBasicTable } from '@elastic/eui';
+
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+interface WorkflowListProps {}
+
+interface WorkflowItem {
+  name: string;
+  description: string;
+}
+
+export function WorkflowList(props: WorkflowListProps) {
+  const [pageIndex, setPageIndex] = useState(0);
+  const [pageSize, setPageSize] = useState(5);
+  const [sortField, setSortField] = useState('name');
+  const [sortDirection, setSortDirection] = useState('asc');
+
+  const columns = [
+    {
+      field: 'name',
+      name: 'Name',
+      sortable: true,
+    },
+    {
+      field: 'description',
+      name: 'Description',
+      sortable: false,
+    },
+  ];
+
+  const items = [
+    {
+      name: 'Workflow 1',
+    },
+  ] as WorkflowItem[];
+
+  const sorting = {
+    sort: {
+      field: sortField,
+      direction: sortDirection,
+    },
+    enableAllColumns: false,
+    readOnly: false,
+  };
+
+  const pagination = {
+    pageIndex,
+    pageSize,
+    totalItemCount: items.length,
```

Comment:
in what scenario this items length could change? it looks like it's a fixed 1 length now as line 36

Reply:
Yeah just a placeholder item for now. We'll fetch the created workflows dynamically just like the AD Detector list page.